lib/rspec/core/formatters/exception_formatter.rb

```diff
@@ -0,0 +1,136 @@
+module RSpec
+  module Core
+    module Formatters
+      # @private
+      class ExceptionFormatter
```

Comment:
Well, it is labeled `@private`, so it's not really intended to be available or visible to users...